hadoop-ozone/dist/src/main/smoketest/s3/objectputget.robot

```diff
@@ -205,3 +205,30 @@ Create file with user defined metadata size larger than 2 KB
     ${custom_metadata_value} =  Execute                    printf 'v%.0s' {1..3000}
     ${result} =                 Execute AWSS3APICli and ignore error        put-object --bucket ${BUCKET} --key ${PREFIX}/putobject/custom-metadata/key2 --body /tmp/testfile2 --metadata="custom-key1=${custom_metadata_value}"
                                 Should not contain                          ${result}   custom-key1: ${custom_metadata_value}
+
+Create small file and expect ETag (MD5) in a reponse header
```

Comment:
Please add test cases where the same file is uploaded with different content (and md5) and the etags is not the same.